pkgs/gnolang/precompile.go

```diff
@@ -90,6 +90,23 @@ func guessRootDir(fileOrPkg string, goBinary string) (string, error) {
 	return rootDir, nil
 }
 
+// GetPrecompileFilenameAndTags returns the filename and tags for precompiled files.
+func GetPrecompileFilenameAndTags(gnoFilePath string) (targetFilename, tags string) {
```

Comment:
Why did you extract this method to the `gnolang` package, when it's a helper used in `gnodev`? Do you plan to use it somewhere else as well?

Reply:
I need this part of the for gno.mod support. Reason why I think it should be part of gnolang package

> @moul I agree with you. Since this pkg has the ability to precompile files. IMHO it should also have the ability to get filename/tags for precompiled files.

Reply:
OK, let's add this.

Please keep opening small PRs when adding new endpoints on official APIs 👍